proto-public/buf.gen.yaml

```diff
@@ -0,0 +1,16 @@
+version: v1
```

Comment:
Meta comment: should `proto-public` be renamed to `proto/public` to align with `proto/private`?

Reply:
Retracting as you no longer have the interim private directory.